From 9690ea4f3b79d07860f3ea55b6fab571721849b5 Mon Sep 17 00:00:00 2001
From: Juan RP <xtraeme@voidlinux.eu>
Date: Thu, 5 May 2016 07:55:57 +0200
Subject: [PATCH] configure.ac: define HAVE_LZMA_STREAM_ENCODER_MT if it is
 detected properly.

... otherwise HAVE_LZMA_STREAM_ENCODER_MT is undefined and the code
for multithreaded xz compression is skipped completely.
---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index 1ab54cb..cb6943f 100644
--- configure.ac
+++ configure.ac
@@ -377,6 +377,9 @@ if test "x$with_lzma" != "xno"; then
       AC_LANG_PROGRAM([[#include <lzma.h>]],
                       [[lzma_stream_encoder_mt(0, 0);]])],
       [ac_cv_lzma_has_mt=yes], [ac_cv_lzma_has_mt=no])])
+  if test "x$ac_cv_lzma_has_mt" != xno; then
+	  AC_DEFINE([HAVE_LZMA_STREAM_ENCODER_MT], [1], [Define to 1 if you have the `lzma_stream_encoder_mt' function.])
+  fi
 fi
 
 AC_ARG_WITH([lzo2],
-- 
2.8.2

